### PR TITLE
Add an example of working/not working code for cluster actor

### PR DESCRIPTION
**Pull request by webfrank**
Just a note on Actor instantiation when using clustering feature.

### CODE REVIEW

**Comment by Tochemey** [comment]
@webfrank I think this is sufficient enough https://github.com/Tochemey/goakt?tab=readme-ov-file#redeployment

**Comment by webfrank** [comment]
> @webfrank I think this is sufficient enough https://github.com/Tochemey/goakt?tab=readme-ov-file#redeployment

On first read it was not clear. Probably adding the example would be clearer.

File: README.md [hunk 1]
```diff
@@ -21,48 +21,52 @@ The project adheres to [Semantic Versioning](https://semver.org) and [Convention
 
 ## Table of Content
 
-- [Design Principles](#design-principles)
```
Comment (Tochemey) [inline README.md:24]: It seems the formatting is wrong. Also I am using tab of 2 for indentation. I think I need to state that in the contribution doc. :)

File: README.md [hunk 2]
```diff
@@ -301,6 +305,28 @@ These methods can be used from the [API](./actors/api.go) as well as from the [P
 This offers simple scalability, partitioning (sharding), and re-balancing out-of-the-box. Go-Akt nodes are automatically discovered. See [Clustering](#clustering).
 Beware that at the moment, within the cluster the existence of an actor is unique.
 
+When levereging cluster re-balancing, leaving the supervisor to re-instatiate an actor in case of node fail, actors should be created as plain structs without parameters. This is required because when the supervisor re-creates needed actors will not be aware of those parameters.
```
Comment (Tochemey) [inline README.md:308]: I don't think this note should be here. It should be rather under `Redeployment` section. Also there is no concept of supervisor in the clustering feature. Once cluster mode is enabled, actors are redeployed whenever the cluster topology changes and the members quorum respected.

**Review by Tochemey — CHANGES_REQUESTED** [review]

**Comment by Tochemey** [comment]
@webfrank I have updated the doc in that regard. I honestly don't think we should add an example for that.

**Comment by Tochemey** [comment]
I closed this PR for lack of activities

**Pull request closed without merging**